CIP-0021/CIP-0021.md

```diff
@@ -0,0 +1,139 @@
+---
+CIP: 0021
+Title: Restrictions on transactions signed by hardware wallets
+Authors: Gabriel Kerekes <gabriel.kerekes@vacuumlabs.com>, Rafael Korbas <rafael.korbas@vacuumlabs.com>, Jan Mazak <jan.mazak@vacuumlabs.com>
+Status: Draft
+Type: Standards
+Created: 2021-06-15
+License: CC-BY-4.0
+---
+
+## Abstract
+
+This CIP describes all the restrictions applicable to Cardano transactions which need to be signed by hardware wallets.
+
+## Motivation
+
+Due to certain limitations of HW wallets, especially very small memory and a limited set of data types supported by Ledger, HW wallets are not able to process all valid transactions which are supported by Cardano nodes.
```

Comment:
```suggestion
Due to certain limitations of hardware (abbrev. HW) wallets, especially very small memory and a limited set of data types supported by Ledger, HW wallets are not able to process all valid transactions which are supported by Cardano nodes.
```

Reply:
Changed in https://github.com/cardano-foundation/CIPs/pull/107/commits/0d287f800481a3ca4cec23caf1a09b0b8f3ef0a4.